.github/workflows/multi-arch-build.yaml

```diff
@@ -0,0 +1,141 @@
+name: build multi-arch images
+
+on:
+  push:
+    branches: [ master ]
+  schedule:
+  - cron:  '0 8 * * *'
+  # allows to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  multi:
+    name: multi-arch Buildah build
+    env:
+      BUILDAH_QUAY_REGISTRY: quay.io/buildah
+      CONTAINERS_QUAY_REGISTRY: quay.io/containers
+      # list of architectures for build
+      PLATFORMS: linux/amd64,linux/s390x,linux/ppc64le,linux/arm64
+
+    # build several images in parallel(upstream and stable)
+    strategy:
+      matrix:
+        # where the Dockerfiles for builds are located at contrib/buildahimage directory
+        source:
+        - upstream
+        - stable
+    runs-on: ubuntu-latest
+    # internal registry for temporary build before push
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+        with:
+          driver-opts: network=host
+          install: true
+
+      - name: Build and push local Buildah
+        uses: docker/build-push-action@v2
+        with:
+          context: contrib/buildahimage/${{ matrix.source }}
+          file: ./contrib/buildahimage/${{ matrix.source }}/Dockerfile
+          platforms: ${{ env.PLATFORMS }}
+          push: true
+          tags: |
+            localhost:5000/buildah/${{ matrix.source }}
+
+      # Simple verification that container works
+      - name: amd64 container sniff test
+        id: sniff_test
+        run: |
+          VERSION_OUTPUT="$(docker run localhost:5000/buildah/${{ matrix.source }} buildah version)"
+          echo "$VERSION_OUTPUT"
+          echo ::set-output name=version_output::$VERSION_OUTPUT
```

Comment:
Uh oh, `$VERSION_OUTPUT` will contain multiple lines, so the "set-output" command will only pick up the first line.  IIRC, handling of newlines isn't implemented well (even with proper quoting of the echo arguments).  Dang...sorry I should have anticipated that :disappointed: 

Perhaps this a problem we can simply ignore for now.